src/patterns/correct-errors/example-error-proto-multiple-error-per-input.njk

```diff
@@ -0,0 +1,96 @@
+
+{% from "components/date-input/_macro.njk" import onsDateInput %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/list/_macro.njk" import onsList %}
+
+{% call
+        onsPanel({
+            title: "There is a problem with this page",
+            type: "error"
+        })
+    %}
+
+        {{
+            onsList({
+                "element": "ol",
+                "itemsList": [
+                    {
+                        "text": "Enter a number between 1 to 12",
+                        "url": "#month-error",
+                        "variants": "inPageLink"
+                    },
+                    {
+                        "text": "Enter a date that is after 1 January 2019",
+                        "url": "#year-error",
+                        "variants": "inPageLink"
+                    }
```

Comment:
These links currently error because `"#month-error"` and `"#year-error"` don't exist on the page

Reply:
missed this

Will update